WordPressVIPMinimum/Sniffs/Security/ProperEscapingFunctionSniff.php

```diff
@@ -26,6 +26,7 @@ class ProperEscapingFunctionSniff extends Sniff {
 	public $escaping_functions = [
 		'esc_url',
 		'esc_attr',
+		'esc_attr__',
```

Comment:
Why add `esc_attr__()` ? And if this should be added, why not the other output escaping functions ?
See: https://developer.wordpress.org/plugins/security/securing-output/

Might also be helpful to use the function names as keys and some form of grouping as the values.

Reply:
I figured the other ones could be added in a separate PR since it doesn't relate to the `esc_attr*()` logic being modified here specifically.

> Might also be helpful to use the function names as keys and some form of grouping as the values.

Noted!